CodeStarterCamp_Week2/LottoWinningChecker.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  LottoNumberChecker.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by JIWOONG on 2023/08/01.
+//
+
+import Foundation
+
+// 당첨번호를 생성하는 함수
+func makeLottoNumber() -> Array<Int> {
+    var temp: Int
+    var lotto: Set<Int> = Set<Int>()
```

Comment:
Set 타입도 Array처럼 empty set을 만들 때 아래와 같이 간단히 나타낼 수 있습니다~!
또한 이름도 명확하게 바꿔보면 좋겠습니다.
```suggestion
    var lottoNumbers: Set<Int> = []
```

* **주의점**
[]로 빈 콜렉션을 선언할 때는 타입 추론(type inference)을 사용할 수 없습니다!
반드시 타입 명시(type annotation)를 사용하여 나타내주어야 합니다 😃

Reply:
주의점을 참고하여 사용하도록 하겠습니다! 🌝